docs/reference/data/understanding-database.md

```diff
@@ -0,0 +1,54 @@
+# Understanding Database
+
+By default, n8n uses SQLite as the database. The credentials, workflows, executions, and webhook are decoupled and store in different tables.
+
```

Comment:
As mentioned in the Jira ticket, can you please also suggest the software that folks can use to open this file? Can you also please add a screenshot of that?